src/gke/cluster_config.list

```diff
@@ -0,0 +1,16 @@
+PROJECT_ID=my-project
+NAME_PREFIX=my-cluster
+ZONE=us-central1-a
+ACCELERATOR_TYPE=nvidia-tesla-a100
+ORCHESTRATOR_TYPE=gke
+NETWORK_CONFIG=default_network
+VM_TYPE=a2-megagpu-16g
+GKE_ENABLE_COMPACT_PLACEMENT=true
```

Comment:
Is this being used anywere?

Reply:
This is the demo file for consumption.

The current flow is running the image directly with a `--env-file` command.

This will change with the 0.7.0 release of Cluster-Provisioning-Tool